State-of-Rhode-Island-eCMS/ecms_profile
# Automatically tag press releases with the current site hostname

## Summary / Approach

Updates the ecms_press_release_publisher module to ensure all press releases are tagged with the current site.
Removes unused classes no longer needed for syndication.
Adds documentation.

## Testing Instruction(s)

Check out branch and test locally using develop environment.
Enable the ecms_press_release_publisher module.
Add/edit one or more press releases, they should all have the "develop-ecms-profile.lndo.site" tag assigned.

## Screenshots

![image](https://user-images.githubusercontent.com/53975352/152593946-7fc0cf7b-c10f-4fc9-86d3-d5fb1b5b58a1.png)

## Metadata

| Question | Answer |
|----------|--------|
| Did you apply meaningful labels to the pull request? | y
| Documentation reflects changes? | y
| `CHANGELOG` reflects changes? | y
| Unit/Functional tests cover changes? | n
| Did you perform browser testing? | y
| Did you provide detail in the summary on how and where to test this branch? | y
| Risk level | low
| Relevant links | https://thinkoomph.jira.com/browse/RIGA-176
